images/macos/tests/Android.Tests.ps1

```diff
@@ -28,10 +26,9 @@ Describe "Android" {
         "tools/proguard",
         "ndk-bundle",
         "cmake",
-        "ndk/$ndkLatestFullVersion",
-        "ndk/$ndkLtsFullVersion",
         $platforms,
         $buildTools,
+        ($ndkFullVersions | ForEach-Object { "ndk/${_}" }),

```

Comment:
nitpick: ($ndkFullVersions | ForEach-Object { "ndk/${_}" }) -> $ndkFullVersions and move `ForEach-Object { "ndk/${_}" ` part at line 12

Reply:
Fixed it